Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:latest
+
+RUN apt-get update && apt-get install -y pdftk texlive-latex-base texlive-latex-extra texlive-fonts-recommended texlive-fonts-extra
+
+WORKDIR /app
+
+RUN pip install --upgrade pip
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /app
+RUN pip install --no-cache-dir -e /app
+
+WORKDIR /build
+
+ENTRYPOINT [ "python", "-m", "dungeonsheets.make_sheets" ]
+CMD [ "--fancy", "--editable", "--recursive" ]
```

Comment:
We declare the default command our container should run.
(In this case we run `python -m dungeonsheets.make_sheets --fancy --editable --recursive`)